.werft/platform-delete-preview-environments-cron.ts

```diff
@@ -54,12 +55,16 @@ class HarvesterPreviewEnvironment {
     // The name of the namespace that the VM and related resources are in, e.g. preview-my-branch
     namespace: string
 
-    // Then name of the preview environment, e.g. my-branch
+    // The name of the preview environment, e.g. my-branch
     name: string
 
+    // The namespace in the k3s cluster where all resources are (default)
+    k3sNamespace: string
```

Comment:
Tiny nit: As this isn't configurable you could set the value here instead of in the constructor.